.changeset/pretty-poems-explode.md

```diff
@@ -0,0 +1,33 @@
+---
+'@keystonejs/fields': major
+'@keystonejs/oembed-adapters': major
+---
+
+Moved `@keystonejs/fields-cloudinary-image`, `@keystonejs/fields-unsplash` and `@keystonejs/fields-oembed` out of the core fields package `@keystonejs/fields`.
+
+```js
+const { CloudinaryImage, Unsplash, OEmbed } = require('@keystonejs/fields');
+```
+
+Needs to be changed to:
+
+```js
+const { CloudinaryImage, Unsplash, OEmbed } = require('@keystonejs/fields-cloudinary-image');
```

Comment:
This line should just be `CloudinaryImage`, right?

Reply:
Yes